src/components/Feature.jsx

```diff
@@ -3,7 +3,7 @@ import { storyblokEditable } from "@storyblok/react/rsc";
 const Feature = ({ blok }) => {
   return (
     <div className="feature" {...storyblokEditable(blok)}>
-      <span>{blok.name}</span>
+      <strong>{blok.name}</strong>
```

Comment:
I would keep it as a span actually, and apply a bold style through CSS.

Reply:
changed